### PR TITLE
build(js): Bump a few more small packages

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1276,6 +1276,17 @@
     "@emotion/utils" "0.11.3"
     "@emotion/weak-memoize" "0.2.5"
 
+"@emotion/cache@^11.1.3":
```
Comment (billyvg) [inline yarn.lock:1279]: Where did these changes come from?

Comment (EvanPurkhiser) [reply yarn.lock:1279]: The @types/refluxbox 0.0.2 bump brings this in :/

We can bump emotion soon here and get these aligned, so I'm not _too_ worried.

File: static/app/components/sentryDocumentTitle.tsx [hunk 2]
```diff
@@ -28,7 +28,11 @@ function SentryDocumentTitle({title, orgSlug, projectSlug, children}: Props) {
 
   const docTitle = getDocTitle();
 
-  return <DocumentTitle title={`${docTitle} - Sentry`}>{children}</DocumentTitle>;
+  return (
+    <DocumentTitle title={`${docTitle} - Sentry`}>
+      {children as React.ReactChild}
```
Comment (priscilawebdev) [inline static/app/components/sentryDocumentTitle.tsx:33]: why do you need to cast children now?

Comment (EvanPurkhiser) [reply static/app/components/sentryDocumentTitle.tsx:33]: We cannot unfortunately, I don't know exactly what the issue is here, but types become incompatible farther up if I do that. I can look closer

File: yarn.lock (hunk 1 above)
Comment (EvanPurkhiser) [reply yarn.lock:1279]: Is this blocking this PR?

Comment (billyvg) [reply yarn.lock:1279]: I would say it's not worth bumping types for this, but if we're planning on upgrading emotion then it's fine

**Review by billyvg — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (EvanPurkhiser) [reply yarn.lock:1279]: I'm just seeing how hard it is to keep everything up to date, though there's probably some tradeoffs there

**Pull request merged**